eng/performance/benchmark_jobs.yml

```diff
@@ -47,6 +49,12 @@ jobs:
             value: ''
           - name: Creator
             value: ''
+          ${{ if eq(parameters.osName, 'windows') }}:
+            - name: HelixPreCommand
+              value: 'set BV_UPLOAD_SAS_TOKEN=$(BV_UPLOAD_SAS_TOKEN)'
+          ${{ if ne(parameters.osName, 'windows') }}:
+            - name: HelixPreCommand
+              value: 'BV_UPLOAD_SAS_TOKEN=$(BV_UPLOAD_SAS_TOKEN)'
```

Comment:
Would you need to export it like in bash (I'm not sure how this is handled in yml/helix)?

Reply:
I am not sure. Possibly. I tried on my mac and just setting it like that was enough, but I expect this might take a small amount of trial and error.

Reply:
Maybe I'll just export it to be safe